MediaBrowser.Model/Updates/PackageInfo.cs

```diff
@@ -9,55 +10,78 @@ namespace MediaBrowser.Model.Updates
     /// </summary>
     public class PackageInfo
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PackageInfo"/> class.
+        /// </summary>
+        public PackageInfo()
+        {
+            Versions = Array.Empty<VersionInfo>();
+            Guid = string.Empty;
+            Category = string.Empty;
+            Name = string.Empty;
+            Overview = string.Empty;
+            Owner = string.Empty;
+            Description = string.Empty;
+        }
+
         /// <summary>
         /// Gets or sets the name.
         /// </summary>
         /// <value>The name.</value>
-        public string name { get; set; }
+        [JsonPropertyName("name")]
+        public string Name { get; set; }
 
         /// <summary>
         /// Gets or sets a long description of the plugin containing features or helpful explanations.
         /// </summary>
         /// <value>The description.</value>
-        public string description { get; set; }
+        /// [JsonPropertyName("description")]
+        public string Description { get; set; }
 
         /// <summary>
         /// Gets or sets a short overview of what the plugin does.
         /// </summary>
         /// <value>The overview.</value>
-        public string overview { get; set; }
+        [JsonPropertyName("overview")]
+        public string Overview { get; set; }
 
         /// <summary>
         /// Gets or sets the owner.
         /// </summary>
         /// <value>The owner.</value>
-        public string owner { get; set; }
+        [JsonPropertyName("owner")]
+        public string Owner { get; set; }
 
         /// <summary>
         /// Gets or sets the category.
         /// </summary>
         /// <value>The category.</value>
-        public string category { get; set; }
+        [JsonPropertyName("category")]
+        public string Category { get; set; }
 
         /// <summary>
-        /// The guid of the assembly associated with this plugin.
+        /// Gets or sets the guid of the assembly associated with this plugin.
         /// This is used to identify the proper item for automatic updates.
         /// </summary>
         /// <value>The name.</value>
-        public string guid { get; set; }
+        [JsonPropertyName("guid")]
+#pragma warning disable CA1720 // Identifier contains type name
+        public string Guid { get; set; }
+#pragma warning restore CA1720 // Identifier contains type name
```

Comment:
I hate to suggest it, but maybe we should change this property to `Id` to avoid all these warnings and use a new location for the next manifest. Just an option.

Reply:
Did mention a new manifest moving forward as we can't roll anything back into 10.6.x to stop that from trying to load 10.7 plugins.

Reply:
You can always change the property name to `Id`, and leave the `JsonPropertyName`  as `guid`

Reply:
Done as suggested